macOSUpgrade.sh

```diff
@@ -110,8 +110,7 @@ userDialog="$9"
 if [[ ${userDialog:=0} != 1 ]]; then userDialog=0 ; fi
 
 ##Title of OS
-##Example: macOS High Sierra
```

Comment:
I think this is not input from jamf parameters. Therefore, I removed this example comment.